CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 18.2.0 (TODO)
+
+### React
+
+* Only treat updates to Suspended components as a new mount in Legacy Mode ([@rickhanlonii](https://github.com/rickhanlonii)) in [#24530](https://github.com/facebook/react/pull/24530)
+
+### React DOM
+
+* Provide a Component Stack in Dev to `onRecoverableError` through an `errorInfo` second argument, similar to `componentDidCatch` in Error Boundaries ([@gnoff](https://github.com/gnoff)) in [#24591](https://github.com/facebook/react/pull/24591)
```

Comment:
We avoid abbreviations like "in Dev" in changelog, let's say "in development". I'd maybe shorten as

- Provide a component stack as a second argument to `onRecoverableError`.